extract.c

```diff
@@ -235,7 +237,14 @@ int isVariant(Config *config, const bam_pileup1_t *plp, uint32_t *coverage, int
     }
 }
 
+int error()
+{
+    printf("fatal: malformed BBM file\n");
+    return -9;
+}
+
 void *extractCalls(void *foo) {
+    //fprintf(stderr, "in extractCalls\n");
```

Comment:
Can you remove this?

Reply:
Thanks, I thought I removed all of these, guess I missed some.